Kernel/CMakeLists.txt

```diff
@@ -404,7 +405,7 @@ if ("${SERENITY_ARCH}" STREQUAL "x86_64")
         # TODO: Share these with the aarch64 build
         Interrupts/SpuriousInterruptHandler.cpp
         kprintf.cpp
```

Comment:
this also looks like a `Library` function (i guess that key to the contents of that folder was different from what i imagine 🤔). maybe other places? (`Boot`!? `Prekernel`?!)

Reply:
I'd like to get this PR merged first, and then we can think about kprintf. Maybe it should be in the TTY folder.